tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "sourceMap": true,
     "target": "ES2019",
     "esModuleInterop": true,
-    "noImplicitAny": true
+    "noImplicitAny": true,
+    "resolveJsonModule": true
```

Comment:
It looks like this is what's causing the [deploy preview](https://deploy-preview-1819--heimdall-lite.netlify.app/) to show a white page instead of the app. Is this necessary to include for the whole app or just hdf-converters?